### ISSUE TITLE
Incorrect normal vector for simplex-based models

### ISSUE THREAD

**Issue by fverdugo** [gridap/Gridap.jl#97]
@santiagobadia this is important to be aware of.

I just found out that the computation of normal vectors is not working for siplex-based models.

I'll will investigate what is going on

Example:

```julia
using Gridap

model = CartesianDiscreteModel(partition=(3,3))
model = simplexify(model)

strian = BoundaryTriangulation(model)
squad = CellQuadrature(strian,order=4)
ns = NormalVector(strian)

q = coordinates(squad)
phi = CellGeomap(strian)

ns_q = evaluate(ns,q)
x = evaluate(phi,q)

writevtk(strian,"strian")
writevtk(x,"x",pointdata=["ns" => ns_q])

```

![bug2](https://user-images.githubusercontent.com/48508177/65031848-4fb76a00-d942-11e9-9b3a-9340ca0062e3.png)

Idem for `SkeletonTriangulation`

![bug1](https://user-images.githubusercontent.com/48508177/65031884-5e9e1c80-d942-11e9-8c40-1e8139ac3eff.png)

**Comment by fverdugo** [comment]
Fixed! Once the changes are in master, I'll close the issue.

The problem was that in formula n=F^{-T}N (n is the physical normal, and N is the reference normal) F is in fact the transpose of the Jacobian J. Thus, the correct formula is n= J^{-1}N, and I was using n=J^{-T}N.

![fixed](https://user-images.githubusercontent.com/48508177/65044748-d62d7500-d95d-11e9-8158-4b6314823344.png)

![fixed2](https://user-images.githubusercontent.com/48508177/65044759-dcbbec80-d95d-11e9-8fd1-26790d122f9e.png)

**Comment by fverdugo** [comment]
Fixed via PR #98 

closing issue
